e2e-tests/package.json

```diff
@@ -29,7 +30,10 @@
     "url": "git+https://github.com/Ontotext-AD/graphdb-workbench.git"
   },
   "devDependencies": {
-    "cypress": "^14.5.0",
+    "@bahmutov/cypress-code-coverage": "^2.7.2",
+    "babel-plugin-istanbul": "^7.0.0",
+    "cross-env": "^7.0.3",
```

Comment:
Can you consider another library here, because cross-env is in maintanance mode since 2020.

Reply:
replaced